src/Admin/AfterActivationSiteScan.php

```diff
@@ -180,6 +202,22 @@ protected function get_amp_compatible_plugins_url() {
 		return 'https://amp-wp.org/ecosystem/plugins/';
 	}
 
+	/**
+	 * Get a URL to AMP compatible themes directory.
+	 *
+	 * For users capable of installing themes, the link should lead to the Theme install page.
+	 * Other users will be directed to the themes page on amp-wp.org.
+	 *
+	 * @return string URL to AMP compatible themes directory.
+	 */
+	protected function get_amp_compatible_themes_url() {
+		if ( current_user_can( 'switch_themes' ) ) {
+			return admin_url( '/theme-install.php?tab=amp-compatible' );
```

Comment:
This link isn't showing any themes:

<img width="969" alt="image" src="https://user-images.githubusercontent.com/134745/156063842-3403060a-f4d6-433a-a6db-5b39e0c0d7f6.png">

This is fixed by renaming `tab` to `browse`:

```suggestion
			return admin_url( '/theme-install.php?browse=amp-compatible' );
```